homeassistant/components/neato/vacuum.py

```diff
@@ -198,6 +258,13 @@ def device_state_attributes(self):
         if self.clean_battery_end is not None:
             data[ATTR_CLEAN_BATTERY_END] = self.clean_battery_end
 
+        if self._robot_map_id is not None:
+            data[ATTR_MAP_ID] = self._robot_map_id
```

Comment:
See comment in earlier PR:
https://github.com/home-assistant/home-assistant/pull/19835#discussion_r247198645

Reply:
Sorry, I forgot to remove that, it's not used actually.

Reply:
@MartinHjelmare Not sure I understand your other comment about registering the service in the platform or the component? Would you happen to have an example?